test/CMakeLists.txt

```diff
@@ -22,4 +22,10 @@ target_link_libraries(lsc_test
 target_link_libraries(lsc_test2
 	-lpthread
 	lazy_sockets
-)
\ No newline at end of file
+)
+
+set(ENV{CTEST_OUTPUT_ON_FAILURE} 1)
+target_compile_features(lsc_test2 PRIVATE cxx_std_17)
```

Comment:
not sure this is actually needed actually... I actually don't know though. It was in the doctests documentation.